lib/manageiq/graphql/types/query.rb

```diff
@@ -21,6 +21,19 @@ module Types
           }
         end
 
+        connection :networks, !Network.connection_type, "List available networks" do
+          argument :tags, types[types.String]
+
+          resolve ->(_obj, args, _ctx) {
+            networks = if args[:tags]
+                         ::CloudNetwork.find_tagged_with(:all => args[:tags].join(" "), :ns => Classification::DEFAULT_NAMESPACE)
```

Comment:
Are all networks going to be via CloudNetwork?  I think having /networks only be CloudNetwork under the covers might be a problem, but maybe not, if we tweak this later.

Reply:
@Fryguy, @agrare, @ladas

I might have had "tunnel vision" so I didn't think about any other types of networks. Are there any other network types? I believe `cloud_networks` encompasses all types including the infra ones, and the type field determines the ems (STI).

Overall that's a good time to review the networks items structure as they're coming across the board. What do you think?

Reply:
@gildub the InfraManager equivalent of a `CloudNetwork` is `Lan`, the `Network` model is more like the ipaddr/hostname/domain for a specific NIC.  Infra doesn't use the `CloudNetwork` model at all.

> Overall that's a good time to review the networks items structure as they're coming across the board. What do you think?

I would :heart: to unify the networking models across cloud/infra/physical eventually

Reply:
@agrare,

Ok great thanks for explanation. I'll rename to CloudNetworks then.

Good luck with the Model re-unification...